spec/features/events_spec.rb

```diff
@@ -79,8 +79,25 @@
       FactoryGirl.create(:application_letter_accepted, :event => @event, :user => @pupil.user)
     end
     visit event_path(@event)
-    click_button I18n.t('events.applicants_overview.sending_acceptances')
-    expect(page).to have_selector('div', :id => 'send-emails-modal')
+    click_link I18n.t('events.applicants_overview.sending_acceptances')
+    fill_in('email_subject', with: 'Subject')
+    fill_in('email_content', with: 'Content')
+    click_button I18n.t('emails.email_form.send')
+  end
```

Comment:
maybe you could test if an email is sent at all? you can find some inspiration in spec/services/mailer_spec.rb:
expect{<action that causes the mail to be sent>}.to change{ActionMailer::Base.deliveries.count}.by(1)
Note that hide_recipients needs to be false in that case, otherwise not 1 but 1 email for each recipient is sent